po/vi.po

```diff
@@ -6421,6 +6463,10 @@ msgid ""
 "Rust docs. Show students the docs for this type, and the helpful link back "
 "to the `keys` method."
 msgstr ""
+"HashMap đi kèm với nhiều kiểu dữ liệu chỉ sử dụng làm kiểu trả về của một số "
+"hàm, như `std::collections::hash_map::Keys`. Những kiểu dữ liệu này thường "
+"xuất hiện trong tài liệu của Rust. Hãy cho học viên xem tài liệu về kiểu dữ "
+"liệu `Keys`, và liên hệ với hàm `keys`."
```

Comment:
Why did you leave "Keys" and "keys" untranslated?

Reply:
`Keys` in this context means the `Keys` type returned by calling method `keys` on a HashMap, documented [here](https://doc.rust-lang.org/std/collections/hash_map/struct.Keys.html), so it's best not to translate them IMO

Reply:
Yeah. I kinda agree with @vohoanglong0107 here.